lib/core/card-view/models/card-view-textitem.model.ts

```diff
@@ -33,23 +33,19 @@ export class CardViewTextItemModel extends CardViewBaseItemModel implements Card
         this.multivalued = !!cardViewTextItemProperties.multivalued;
         this.pipes = cardViewTextItemProperties.pipes || [];
         this.clickCallBack = cardViewTextItemProperties.clickCallBack ? cardViewTextItemProperties.clickCallBack : null;
-    }
 
-    get displayValue() {
-        if (this.isEmpty()) {
-            return this.default;
-        } else {
-            return this.applyPipes(this.value);
+        if (cardViewTextItemProperties.default && this.isEmpty()) {
+            this.value = cardViewTextItemProperties.default;
         }
     }
 
-    private applyPipes(displayValue) {
+    get displayValue() {
         if (this.pipes.length) {
-            displayValue = this.pipes.reduce((accumulator, { pipe, params = [] }) => {
+            this.value = this.pipes.reduce((accumulator, { pipe, params = [] }) => {
```

Comment:
is there any way to optimise/cache the result and avoid this code on each rendering cycle?